src/model/reduction.h

```diff
@@ -27,6 +28,7 @@ class PotForce {
 public:
 	real_t E;
 	vec3_t pos;
+	real_t modes[10];
```

Comment:
see comment in ""Types_6d_Modes.h": do not hard-code the "10"
I think we need to distinguish between a PotForce class as it was, and a new "PotForceModes". Maybe a new file for those?

Reply:
was hälst du davon datei potforcetypes.h zu erstellen und eine reduction_modes.h. 

die potforcetypes.h enthält die types für und ohne moden und die reduction_modes.h hat die included reduction.h und enthält noch alle funktionen die unique für die mode reduction sind. 

So hätte man die moden und nicht moden klar getrennt und doppelten code vermieden.